.pubnub.yml

```diff
@@ -103,6 +115,21 @@ features:
     - UNSUBSCRIBE-ALL
   signal:
     - SIGNAL-SEND
+  objects:
```

Comment:
Missing `minor` version bump.

Reply:
The version was increased from `1.1.0` to `1.2.0`